docs/using-mycroft-ai/customizations/languages/german.md

```diff
@@ -0,0 +1,286 @@
+---
+description: >-
+  Dies ist eine Anleitung wie Mycroft in Deutsch eingerichtet werden kann. Diese Seite beinhaltet auch eine Kurzanleitung.
+---
+   <sub><sub>TTS = Sprachoutput  / STT = Sprachinput    / WakeWord = Initiierung </sub></sub>
+---
+
+# Mycroft auf deutsch
+
+## Erster Einstieg
+
+!!!Aktuell ist das paaren von Mycroft noch nicht vollständig in deutsch möglich, daher sollten sie Mycroft bereits unter
+"lang": "en-us" verbunden und eingerichtet haben.
+
+Zuerst, [verbinden Sie sich mit ihrem Gerät (z.B. SSH)](https://mycroft.ai/documentation/mark-1/#connecting-to-the-mark-1-via-ssh).
+
+Folgend wird an den Beispielen Google/Mbrola der Sprachoutput (TTS) auf deutsch gesetzt. Mycroft bietet weitere TTS-Möglichkeiten über [marytts](http://mary.dfki.de/), [responsive_voice](https://responsivevoice.org/), [polly](https://aws.amazon.com/de/polly/), auf die hier jedoch nicht näher eingegangen wird.
+
+## Text-to-Speech (TTS) über Google
+
+
+Erstelle oder ändere die eigene `~/.mycroft/mycroft.conf` Konfigurationsdatei wie folgt.
+
+**Data:**
+
+ ```javascript
+{
+  "lang": "de-de",
+  "tts": {
+    "module": "google",
+    "google": {
+      "lang": "de"
+    }
+  }
+}
+
+```
+_HINWEIS: Sollte sich die Standardausgabe von Mycroft in deutsch melden hilft eventuell das updaten mit `mycroft-pip install --upgrade gtts` oder `mycroft-pip install --upgrade gtts-token`._
+
+## TTS Alternativ mit espeak mbrola
+
+Für Rasberry kann mbrola hier herunterladen werden mit `wget http://steinerdatenbank.de/software/mbrola3.0.1h_armhf.deb` und `sudo dpkg -i mbrola3.0.1h_armhf.deb`.
+Installiere nun espeak mbrola über `sudo apt install espeak espeak-data mbrola mbrola-de7`.
+
+**Data:**
+
+```javascript
+{
+  "lang": "de-de",
+  "tts": {
+    "module": "espeak",
+    "espeak": {
+      "lang": "german-mbrola-7",
+      "voice": "german-mbrola-7"
+    }
+  }
+}
+```
+_HINWEIS: Die Stimme 7 kann auch durch eine andere Stimme ersetzt werden. Dabei sind alle geraden Zahlen männlich und alle ungeraden Stimmen weiblich._
+
+Mycroft sollte nach einem Neustart auf deutsch hören und sprechen. Viele Skills sind bereits übersetzt und können sofort genutut werden.
+
+Das **Wake Word** (z.b. hey mycroft, Christopher, Hey Ezra, Hey Jarvis) wird hierbei unter [Mycroft Home](https://home.mycroft.ai) eingerichtet.
+
+# Ändern des **Wake Words** in ein deutsches Wort
```

Comment:
Hey, we should only ever have 1 top level heading (h1) per page.

Can we make this a h2 `##` and add more levels to the headings below it if needed?

Reply:
This is also the case, down lower